core/templates/dev/head/pages/exploration_player/RefresherExplorationConfirmationModalService.js

```diff
@@ -59,7 +59,7 @@ oppia.factory('RefresherExplorationConfirmationModalService', [
                 // TODO(bhenning): Find a reliable way to send events that does
                 // not get interrupted with browser redirection.
                 $timeout(function() {
-                  $window.open(url, '_self');
+                  //$window.open(url, '_self');
```

Comment:
?

Reply:
Yeah, did that for testing, saw that just after I pushed, removed that.